integration/test/README.md

```diff
@@ -44,19 +26,37 @@ application to wrap target job execution.
               $PATH_TO_DDT_CLIENT_SESSION_FILE" \
               ./geopm_test_integration.py -v TestIntegration.test_count`
 
-EXAMPLES
---------
-Since the log files that are emitted from the individual tests are
-opened and appended to on subsequent runs, it may become necessary to
-periodically clean up the files to avoid conflating experimental data.
-```git clean``` can be used in this instance to clean up all of the
-temporary files:
-
-    $ git clean -fd .
-    $ export GEOPM_RUN_LONG_TESTS=true
-    $ ./geopm_test_integration.py -v TestIntegration.test_power_consumption
-
-Or as a one-liner:
-
-    $ git clean -fd . ; GEOPM_RUN_LONG_TESTS=true ./geopm_test_integration.py -v TestIntegration.test_power_consumption
-
+EXAMPLE
+-------
+
+```
+#!/bin/bash
+
+# Set to geopm source code location
+export GEOPM_SOURCE=/home/test/Git/geopm
+
+# Set to the install prefix provided to configure
+export GEOPM_INSTALL=/home/test/build/geopm
+
+# Find location of geopmdpy site-packages
+export GEOPM_PYTHONPATH=$(dirname $(find ${GEOPM_INSTALL} -name geopmdpy -type d))
```

Comment:
This is different than the code that's already in `run_env.sh` that accomplishes the same thing.  We should only have it in one spot.

Reply:
This is a README with instructions on how to use our integration tests...I'm not sure it is helpful to refer to bash script...